soda/core/tests/cli/test_cli_update_distribution_file.py

```diff
@@ -2,15 +2,20 @@
 
 import pytest
 from cli.run_cli import run_cli
-from helpers.common_test_tables import customers_test_table
+from helpers.common_test_tables import (
+    customers_test_table,
+    dro_categorical_test_table,
+    null_test_table,
+)
 from helpers.data_source_fixture import DataSourceFixture
 from helpers.fixtures import test_data_source
 from helpers.mock_file_system import MockFileSystem
+from ruamel.yaml import YAML
 
 
 @pytest.mark.skipif(
-    test_data_source != "postgres",
-    reason="Run for postgres only as nothing data source specific is tested.",
+    test_data_source not in ["postgres", "snowflake", "bigquery", "mysql"],
+    reason="Other data sources are not supported",
```

Comment:
why we don't support other data sources? We just do `select * from where or group by` right?

Reply:
Yeah I don't know why actually, I'll have a look to see how hard it would be to fix!